lib/kaper.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Kaper do
+
+  def start do
+    ensure_started HTTPoison
```

Comment:
In fact you don't need that function just use
`:application.ensure_all_started(:httpoison)`